docs/docs/contributing-guide/marketplace/marketplace-setup.md

```diff
@@ -0,0 +1,65 @@
+---
+id: marketplace-setup
+title:  'Marketplace: Setup'
+---
+
+The Marketplace offers custom plugins that can be installed in your ToolJet instance. This guide aims to assist you in creating a new plugin for the ToolJet marketplace.
```

Comment:
It may be more appropriate to use a title such as "Marketplace Development Setup" for a guide that covers the initial setup steps for developing plugins, rather than simply "Marketplace: Setup".